src/config.rs

```diff
@@ -2351,6 +2345,31 @@ impl ConfigHandler {
     }
 }
 
+use crate::raftstore::store::DynamicConfig;
+impl DynamicConfig for ConfigHandler {
+    fn refresh(&mut self, cfg_client: &dyn ConfigClient) {
+        debug!(
+            "refresh config";
+            "component id" => self.get_id(),
+            "version" => ?self.get_version()
+        );
+        if let Err(e) = self.refresh_config(cfg_client) {
+            warn!(
+                "failed to refresh config";
+                "component id" => self.get_id(),
+                "version" => ?self.get_version(),
+                "err" => ?e
+            )
+        }
+    }
+    fn refresh_interval(&self) -> Duration {
+        self.get_refresh_interval()
+    }
+    fn get(&self) -> String {
+        toml::to_string(self.get_config()).unwrap()
```

Comment:
Maybe we should return `&TiKvConfig` and let the caller decide what format to use.

Reply:
`TiKvConfig` is defined `tikv::config`, changing to TiKvConfig causes cycling depend problem.

Reply:
OK